### PR TITLE
Fix up and modify logstash filters for new environment

### PR DESCRIPTION
**Pull request by d34dh0r53**
This commit modifies existing logstash filters to work with the updated
logging stack.  This patch creates a new field called servicemessage
where the requestid and additional fields are stripped from a meaningful
message.

Connects rcbops/u-suk-dev#235

### CODE REVIEW

**Comment by Apsu** [comment]
RPC-AIO-Liberty-PR: Fail

**Comment by Apsu** [comment]
RPC-AIO-Liberty-PR: Fail

**Comment by Apsu** [comment]
RPC-AIO-Liberty-PR: Fail

**Comment by Apsu** [comment]
RPC-AIO-Ceph-Liberty-PR: Fail

**Comment by Apsu** [comment]
RPC-AIO-Ceph-Liberty-PR: Fail

**Comment by Apsu** [comment]
RPC-AIO-Ceph-Liberty-PR: Fail

**Comment by Apsu** [comment]
RPC-AIO-Ceph-Liberty-PR: Fail

**Comment by Apsu** [comment]
RPC-AIO-Liberty-PR: Fail

**Comment by Apsu** [comment]
RPC-AIO-Ceph-Liberty-PR: Fail

**Comment by Apsu** [comment]
RPC-AIO-Liberty-PR: Fail

File: rpcd/playbooks/roles/logstash/tasks/logstash_post_install.yml [hunk 1]
```diff
@@ -73,6 +73,7 @@
     - 17-rabbitmq.conf
     - 18-ceph.conf
     - 19-nginx.conf
+    - 98-traceback.conf
```
Comment (git-harry) [inline rpcd/playbooks/roles/logstash/tasks/logstash_post_install.yml]: The checks are failing with
```
fatal: [jrpcaioiad-5c1_logstash_container-c5f45846] => input file not found at /jenkins/workspace/RPC-AIO/rpcd/playbooks/roles/logstash/templates/98-traceback.conf or /jenkins/workspace/RPC-AIO/rpcd/playbooks/98-traceback.conf
```
It looks like the template wasn't added to the commit.

**Comment by d34dh0r53** [comment]
@git-harry thanks, yeah, that file didn't get included in my diff and was missed.  I've added it now.

File: rpcd/playbooks/roles/filebeat/defaults/main.yml [hunk 2]
```diff
@@ -50,6 +50,7 @@ filebeat_logstash_hosts: "{% for host in groups['logstash_all'] %}{{ hostvars[ho
 #
 #                              Date        Time       PID     Level   Python Module   Id
 multiline_openstack_pattern: '^[0-9-]{10} +[0-9:\.]+ +[0-9]+ +[A-Z]+ +[A-Za-z0-9\._]+ \[|Traceback'
+multiline_keystone_error_pattern: '^[0-9-]{10} +[0-9:\.]+ [0-9-]{10} +[0-9]+ +[A-Z]+ +[A-Za-z0-9\._]+ \[|Traceback'
```
Comment (sigmavirus24) [inline rpcd/playbooks/roles/filebeat/defaults/main.yml]: Looking at [this log event](https://104.239.156.122:8443/app/kibana#/doc/logstash-*/logstash-2016.08.09/log?id=AVZwal3bXLZuH-GTFFB4&_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:now-15m,mode:quick,to:now))) this doesn't seem to work

**Comment by sigmavirus24** [comment]
This needs another round of review but I'm confident this works exactly as @d34dh0r53 intended now.

**Comment by d34dh0r53** [comment]
:+1: on @sigmavirus24's change, looks good.

**Comment by sigmavirus24** [comment]
I have an AIO with this running but I can't seem to get Keystone to produce tracebacks in the keystone-apache-error.log file. =/

**Comment by sigmavirus24** [comment]
Even though I contributed to this patch, it was a minor portion and I was able to confirm the rest of it worked fine even if I can't confirm the bit that I've contributed at the moment outside of the golang link (and those experiments have proven valuable and properly representative in the past).

👍

**Comment by sigmavirus24** [comment]
Apologies if a minor contributor shouldn't have merged the patch they contributed to, but the CONTRIBUTING guidelines aren't clear on this situation.

**Comment by hughsaunders** [comment]
Posthumous :+1

**Pull request merged**